CMakeLists.txt

```diff
@@ -231,6 +231,16 @@ if (WIN32)
             string(REPLACE "/INCREMENTAL" "/INCREMENTAL:NO" ${LinkerFlag} ${${LinkerFlag}})
         endforeach()
     endif()
+
+    if (MSVC)
+        option(FILAMENT_SHORTEN_MSVC_COMPILATION "Shorten compile-time in Visual Studio" ON)
```

Comment:
would it make sense to always do this for MSVC? In which case we wouldn't need the extra option.